src/frontend/src/pages/app/subroutes.tsx

```diff
@@ -0,0 +1,11 @@
+import { Route } from "react-router-dom";
+import NoGraph from "~/pages/app/no-graph";
+import GraphNotFound from "./graph-not-found";
+
+export function getAppRoutes() {
+    return <>
+        <Route index element={<NoGraph />} />
```

Comment:
```suggestion
        <Route index element={<GraphLandingPage />} />
```
`NoGraph` finde ich gefährlich ähnlich zu `GraphNotFound`. Dacher fände ich das, "DefaultGraph", oder "GraphSelection" ein besseres naming

Reply:
mach ich beim nächsten update